studybuddies/src/app/components/login/login.component.ts

```diff
@@ -7,9 +12,44 @@ import { Component, OnInit } from '@angular/core';
 })
 export class LoginComponent implements OnInit {
 
-  constructor() { }
+  user: User = new User();
+  loginError = null;
+  logged: boolean;
 
-  ngOnInit(): void {
+  constructor(private loginService: LoginService, public router: Router, private cookieService: CookieService, public translate:TranslateService) { }
+
+  ngOnInit() {
+    if (this.cookieService.get('auth_token') != "") {
+      this.logged = true;
+      this.loginService.getPrincipal()
+        .then (user => {
+          if(user.tutor == true)
+            this.router.navigate(['/tutor']);
+          else
+            this.router.navigate(['/student']);
+        });
+    } else
```

Comment:
Codacy has a fix for the issue: [Expected { after 'else'.](https://app.codacy.com/gh/Team-Studybuddies/Studybuddies-Frontend/pullRequest?prid=7199829)
```suggestion
    } else
```